task.sh

```diff
@@ -6,38 +6,40 @@
 #   dir2
 #   dir3
 #       dir4
-
+mkdir task task/dir1 task/dir2 task/dir3 task/dir3/dir4
```

Comment:
Если использовать флаг `-p`, то можно не указывать промежуточные папки
```suggestion
mkdir -p task/dir1 task/dir2 task/dir3/dir4
```